packages/react-three-create/src/index.ts

```diff
@@ -162,13 +166,12 @@ export function generate(options: GenerateOptions) {
   generateZustand(generator, clonedOptions.zustand)
   generateFiber(generator, clonedOptions.fiber)
   generateGithubPages(generator, clonedOptions.githubPages)
+  generateTriplex(generator, clonedOptions.triplex)
```

Comment:
It's important for Triplex to come last so it knows what has been selected in options.